Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -23,6 +23,7 @@
       - id: LunchboxCommandFilledRandom # Delta-V Lunchboxes!
         prob: 0.3
       - id: WeaponEnergyGunMini #LPP
+      - id: LPPQMPanties #LPP
```

yamllint

[warning] too few spaces before comment

(comments)